forge-gui/res/cardsfolder/upcoming/the_master_multiplied.txt

```diff
@@ -0,0 +1,9 @@
+Name:The Master, Multiplied
+ManaCost:4 B R
+Types:Legendary Creature Time Lord Rogue
+PT:4/3
+K:Myriad
+S:Mode$ IgnoreLegendRule | ValidCard$ Creature.YouCtrl+token | Description$ The "legend rule" doesn't apply to creature tokens you control.
+S:Mode$ CantSacrifice | ValidCard$ Creature.YouCtrl+token | ValidCause$ Triggered.YouCtrl | ForCost$ False | Description$ Triggered abilities you control can't cause you to sacrifice or exile creature tokens you control.
+S:Mode$ CantExile | ValidCard$ Creature.YouCtrl+token | ValidCause$ Triggered.YouCtrl | ForCost$ False | Secondary$ True | Description$ Triggered abilities you control can't cause you to sacrifice or exile creature tokens you control.
```

Comment:
If you want to keep the check in `ChangeZoneEffect.removeFromStack()` as future-proofing we might have to add `+inZoneBattlefield` here:
this is because PlayEffect CopyCard$ sets it to token early (which I think fixed some other bug)
this means right now e. g. _Ashiok's Erasure_ wouldn't work completely against a spell from _Reenact the Crime_